index.html

```diff
@@ -140,10 +156,285 @@ <h2>Website Protfolio</h2>
               <li>Bootstrap</li>
               <li>Ruby</li>
             </ul>
+            <button data-model-target="#model6" class="button">
+              See Project
+            </button>
           </div>
         </div>
       </section>
 
+      <div class="model" id="model">
+        <div class="model-header">
+          <h2 class="model-title">
+            Keeping track of hundreds of components website
+          </h2>
+          <button data-close-button class="close-button">&times;</button>
+        </div>
+        <ul class="server">
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+        </ul>
+        <div class="model-body">
+          <img src="./images/Snapshoot-Portfolio.png" alt="Project picture" class="model-img">
+          <div class="model-content">
+            <p class="model-paragraph">
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it 1960s with the releaLorem Ipsum is simply
+              dummy text of the printing and typesetting industry. Lorem Ipsum
+              has been the industry's standard dummy text ever since the 1500s,
+              when an unknown printer took a galley of type and scrambled it
+              1960s with the releax map lapora verita.
+            </p>
+            <div class="model-button">
```

Comment:
- [ ] Great effort working on your pop-up modal, 👋 👋  But according to the project requirement it was required of us to create the `project cards section` of your portfolio `dynamically` in `JavaScript` using the `array of an object` method

   > N.B: ![js](https://user-images.githubusercontent.com/78634841/211175508-4cb62c72-6182-47a2-b724-6cda71b69f8f.png)

Example 👍 

- You can check this out for your peruse [Link](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and this [Link](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96.)


- N.B: both the project card and the pop-up cards should be created dynamically using JavaScript